scripts/start_online_demo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
I would consider to make this script accept an argument to specify the host:port. If port is missing, then we default to 80. So from the render console, we specify:

sh /usr/local/bin/start_online_demo.sh https://demo.bytebase.com

And we rename this script to scripts/demo.sh, so that we are actually adding a demo running feature to this image.

Reply:
Also we need to mention who current uses this. (e.g. mention how render uses it)